Lib/logging/handlers.py

```diff
@@ -1080,6 +1081,7 @@ def emit(self, record):
             import smtplib
             from email.message import EmailMessage
             import email.utils
+            import ssl
```

Comment:
It's probably worth moving this import to after the `if self.secure is not None:` line so it only affects performance in those cases; `ssl` is an expensive import.